### DISCUSSION TITLE
startPlayer not working  on iOS with specific URLs and 3.x.x version

### DISCUSSION THREAD

**Discussion by ChawkiOS** [dooboolab/react-native-audio-recorder-player#479] [Q&A]
### Version of react-native-audio-recorder-player
`"react-native-audio-recorder-player": "^3.5.1"`

### Version of React Native
`"react-native": "0.66.0"`

### Platforms you faced the error (IOS or Android or both?)
`IOS`

### Expected behavior
`startPlayer` works with an audio I stored in my server.
```
const path = 'https://staging.media.ensembl.fr/original/uploads/26403543-c7d0-4d44-82c2-eb8364c614d0'
await audioPlayer.startPlayer(path);
```

### Actual behavior
It only works on `Android` with `"react-native-audio-recorder-player": "^3.5.1"`
It works on `iOS` and `Android` when I downgrade to `"react-native-audio-recorder-player": "^2.7.0"`

**Comment by hyochan** [comment]
@ChawkiOS Hello thanks for raising this issue!

I've tested this with the `.mp3` extension and this works nice. You can try with the below URL.
```
https://firebasestorage.googleapis.com/v0/b/cooni-ebee8.appspot.com/o/test-audio.mp3?alt=media&token=d05a2150-2e52-4a2e-9c8c-d906450be20b
```
Related: https://stackoverflow.com/a/15850580/8841562

The reason `react-native-audio-recorder-player@2+` worked is that this does not stream the audio from the URL. Rather it downloads the file first and then plays. Related issue in #231

**Comment by hyochan** [comment]
@ChawkiOS Try to look into #480 either
